infra/indexer/components/market-indexer.ts

```diff
@@ -22,6 +22,8 @@ export interface MarketIndexerArgs {
   dockerRemoteBuilder?: string;
   orderStreamUrl?: pulumi.Output<string>;
   orderStreamApiKey?: pulumi.Output<string>;
+  bentoApiUrl: pulumi.Output<string>;
+  bentoApiKey: pulumi.Output<string>;
```

Comment:
Lets make these optional so its possible to run an indexer without doing execution (e.g. if wanting to do a dev deployment to test)